### PR TITLE
auras: Element update

### PR DESCRIPTION
**Pull request by ls-**

### CODE REVIEW

File: elements/auras.lua [hunk 1]
```diff
@@ -46,7 +46,7 @@ At least one of the above widgets must be present for the element to work.
 
 ## Attributes
 
-button.owner    - the unit who cast the aura (string)
+button.caster   - the unit who cast the aura (string)
```
Comment (p3lim) [inline elements/auras.lua]: Document this on WoWI

File: elements/auras.lua [hunk 2]
```diff
@@ -32,6 +32,7 @@ At least one of the above widgets must be present for the element to work.
 
 .numBuffs     - The maximum number of buffs to display. Defaults to 32 (number)
 .numDebuffs   - The maximum number of debuffs to display. Defaults to 40 (number)
+.numTotal     - The maximum number of auras to display. Defaults to sum of .numBuffs and .numDebuffs (number)
```
Comment (p3lim) [inline elements/auras.lua]: Document this on WoWI

File: elements/auras.lua [hunk 3]
```diff
@@ -15,6 +15,7 @@ At least one of the above widgets must be present for the element to work.
 
 ## Options
 
+.disableMouse       - Disables mouse events (boolean)
```
Comment (p3lim) [inline elements/auras.lua]: Document this on WoWI

File: elements/auras.lua (hunk 2 above)
Comment (Rainrider) [inline elements/auras.lua]: Defaults to **_the_** sum

Comment (Rainrider) [inline elements/auras.lua]: Lacks documentation that this would prioritize buffs to debuffs.

**Pull request merged**